templates/shared_services/sonatype-nexus-vm/scripts/deploy_nexus_container.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+docker_pull_timeout=30
```

Comment:
This equates to 30 minutes? Is that too long?

Reply:
Yes, will reduce to 10 minutes.

Reply:
Resolved with [e667595](https://github.com/microsoft/AzureTRE/pull/3676/commits/e667595de8720affc9461fb6d171f7bd0f0c047f)